calour/database.py

```diff
@@ -383,6 +404,7 @@ def enrichment(self, exp, features, *args, **kwargs):
     def show_term_details(self, term, exp, features, *args, **kwargs):
         '''
         Show details about the specific term in the database and in what features it appears
+        This is an optional function for databases that support enrichment analysis (level L3).
```

Comment:
this does not looks like for enrichment...

Reply:
Improved the explanation. This function is used from the enrichment gui, so usually called from L3 integration.